### ISSUE TITLE
Chunk save Error

### ISSUE THREAD

**Issue by MagmaTeam** [IzzelAliz/Arclight#745]
### I have confirmed that ...

- [X] Arclight is up to date
- [X] all dependencies are installed
- [X] all plugins and mods are up to date
- [X] unable to reproduce in Spigot
- [X] unable to reproduce in Forge

### Arclight version

arclight-1.16.5-1.0.24-SNAPSHOT-5ada7754

### OS & Java versions

Java 11

### Plugins and Mods

```raw
[18:17:54] [Server thread/INFO] [Console/]: ------------=[ ServerUtils Plugins ]=-------------
[18:17:54] [Server thread/INFO] [Console/]:  Plugins (37): BetterRTP, BuycraftX, CoralStay, CoreProtect, DeluxeMenus, DiscordSRV, Essentials, EssentialsChat, EssentialsDiscord, EssentialsEconomyBridge, EssentialsSpawn, FastAsyncWorldEdit, FastChunkPregenerator, GriefDefender, HolographicDisplays, InteractiveBoard, InteractiveChat, LiteBans, LuckPerms, MarriageMaster, Multiverse-Core, MyCommand, PVPToggle, PlaceholderAPI, PlayerKits, ProtocolLib, RewardsLite, ServerUtils, ShopGUIPlus, ShopGUIPlus-SellGUI, SimpleVoteListener, TabList, Vault, VoidGenerator, Votifier, WorldBorder and spark
```

### Description

Having this chunk save error, https://mcpaste.io/c394ad8060d7547b

Could also be apart of the reason why sense the update, my armor stands keep deleting as well as what is in item frames. 

### Step to reproduce

Basically is just always spamming console.

### Logs

https://www.dropbox.com/s/4ei9tnu5xrryojb/latest.zip?dl=1
Ubuntu Is down and file drag didnt work so its in a zip on dropbox^^

**Comment by IzzelAliz** [comment]
Fixed in ef6b96f24a693a54f7f172c2d4313bae7879c176.
